extensions/wikia/StoriesLinkTag/StoriesLinkTagController.class.php

```diff
@@ -140,7 +140,8 @@ public function render() {
 		$urls = array_slice( array_filter( $urls ), 0, self::MAX_URLS );
 		foreach ( $urls as $url ) {
 			$urlParts = parse_url( trim( $url ) );
-			if ( !empty( $urlParts['host'] ) && strtolower( $urlParts['host'] ) == StoriesLinkTagHelper::VALID_HOST
+			if ( !empty( $urlParts['scheme'] ) && strtolower( $urlParts['scheme'] ) == 'http'
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1067)